controllers/endpoints.controller.js

```diff
@@ -0,0 +1,7 @@
+const { selectAllEndpoints } = require("../models/index");
+
+exports.getEndpoints = (req, res) => {
+  selectAllEndpoints().then((endpoints) => {
+    res.status(200).send(endpoints);
```

Comment:
send your endpoints in an object so you can give it a key that makes more sense than just 'body'